x/ibc/core/exported/client.go

```diff
@@ -32,6 +32,14 @@ type ClientState interface {
 	Validate() error
 	GetProofSpecs() []*ics23.ProofSpec
 
+	// Initialization function
+	// Clients must validate the initial consensus state, and may store any client-specific metadata
+	// necessary for correct light client operation
+	Initialize(sdk.Context, codec.BinaryMarshaler, sdk.KVStore, ConsensusState) error
+
+	// Genesis function
+	ExportMetadata(sdk.KVStore) []GenesisMetadata
```

Comment:
Need this function to correctly export client metadata like tendermint's processed times since client keeper doesn't know where in client store these are stored. But once provided by the individual clients, client keeper can easily export and re-import them.